front/src/modules/trainschedule/components/Timetable/TimetableToolbar.tsx

```diff
@@ -194,21 +195,33 @@ const TimetableToolbar = ({
     }
   };
 
-  // TODO Paced train : Adapt this to handle export paced trains in issue https://github.com/OpenRailAssociation/osrd/issues/10614
-  const exportTrainSchedules = (selectedTrainIdsFromClick: TimetableItemId[]) => {
+  const exportTimetableItems = (selectedTimeTableIdsFromClick: TimetableItemId[]) => {
     if (!timetableItems) return;
 
-    const formattedTrainSchedules = timetableItems
-      .filter(({ id }) => isTrainSchedule(id) && selectedTrainIdsFromClick.includes(id))
-      // eslint-disable-next-line @typescript-eslint/no-unused-vars
-      .map(({ id, timetable_id, ...trainSchedule }) => trainSchedule);
+    const formattedTimetableItems = timetableItems
+      .filter(({ id }) => selectedTimeTableIdsFromClick.includes(id))
+      .reduce<{
+        train_schedules: TimetableItem[];
+        paced_trains: TimetableItem[];
```

Comment:
Should this rather be this ?
```suggestion
        train_schedules: TrainScheduleBase[];
        paced_trains: PacedTrain[];
```

Reply:
Done

Reply:
Done